src/Microsoft.IdentityModel.JsonWebTokens/JsonWebTokenHandler.ValidateToken.cs

```diff
@@ -511,6 +513,10 @@ await ValidateJWEAsync(jsonWebToken, validationParameters, currentConfiguration)
                     // where a new valid configuration was somehow published during validation time.
                     if (currentConfiguration != null)
                     {
+                        _telemetryClient.IncrementOperationCounter(
```

Comment:
Consider hiding the implementation inside Microsoft.IdentityModel.Logging.Logger.
Instead of each class having a telemetryClient, expose a method on Logger something like: Logger.IncrementOperationCounter(TelemetryConstants.LKG).

In this way when we expand telemetry into other classes, the adoption pattern is simpler.